circus-train-core/src/main/java/com/hotels/bdp/circustrain/core/CopierFactoryManager.java

```diff
@@ -55,13 +56,21 @@ void postConstruct() {
     }
   }
 
-  CopierFactory getCopierFactory(Path sourceLocation, Path replicaLocation) {
+  CopierFactory getCopierFactory(Path sourceLocation, Path replicaLocation, Map<String, Object> copierOptions) {
```

Comment:
Please follow our formatting if possible makes it just that bit easier to review :)